README.md

```diff
@@ -46,7 +46,7 @@ Code like this:
     "imports": {
       "three": "https://cdn.jsdelivr.net/npm/three@0.164.1/build/three.module.js",
       "three/addons/": "https://cdn.jsdelivr.net/npm/three@0.164.1/examples/jsm/",
-      "@pixiv/three-vrm": "three-vrm.module.js"
+      "@pixiv/three-vrm": "three-vrm.mjs"
```

Comment:
we also need to change this:

https://github.com/pixiv/three-vrm/pull/1411/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R36